src/bfs.c

```diff
@@ -0,0 +1,49 @@
+#include "bfs.h"
+
+// Function to initialise the attributes of each node
```

Comment:
Regarding this.
I think one line commenting should be done with `//` as this will add a bit more flexiblility